gamemodes/terrortown/gamemode/shared/lang/polish.lua

```diff
@@ -0,0 +1,1222 @@
+-- English language strings
+
+local L = LANG.CreateLanguage("Polski")
+
+-- General text used in various places
+L.traitor    = "Zdrajca"
+L.detective  = "Detektyw"
+L.innocent   = "Niewinny"
+L.last_words = "Ostatnie słowa"
+
+L.terrorists = "Terroryści"
+L.spectators = "Obserwatorzy"
+
+L.noteam = "BRAK TEAMU"
+L.innocents = "TEAM Niewinni"

```

Comment:
Shouldn't this be TEAMU? or BRAK?

Reply:
It means no(or better none) team, it is good like it is